frontend/packages/console-app/locales/ko/console-app.json

```diff
@@ -402,14 +480,22 @@
   "Close": "닫기",
   "Back": "이전",
   "Restart": "재시작",
-  "In this quick start, you will complete {{count, number}} task_0": "이 퀵스타트에서는 {{count, number}} 작업을 완료합니다",
+  "In this quick start, you will complete {{count, number}} task_one": "이 퀵스타트에서는 {{count, number}} task_one 작업을 완료합니다",
+  "In this quick start, you will complete {{count, number}} task_other": "이 퀵스타트에서는 {{count, number}} task_other 작업을 완료합니다",
```

Comment:
@cyril-ui-developer I don't really understand the background of why there were keys with the suffix `_0` for non-en languages, so I'm not really sure why there was only 1 version of this string for Korean. Here, "task_one" and "task_other" have been added into the strings themselves, which must be a mistake?

Perhaps @rh-tchuang could help here.

Reply:
> @cyril-ui-developer I don't really understand the background of why there were keys with the suffix `_0` for non-en languages, so I'm not really sure why there was only 1 version of this string for Korean. Here, "task_one" and "task_other" have been added into the strings themselves, which must be a mistake?
> 
> Perhaps @rh-tchuang could help here.

@KevinFCormier I don't know either. However the APAC languages(zh,ko and ja) at this stage don't support plurals.  In this test batch, the translators translated for ja and zh only,  manually ignoring the suffix _one and _other which could be prone to human error. Korean was not translated in this batch. that is why you are seeing  the suffix _one and _other.